tests/test_app.py

```diff
@@ -2,6 +2,7 @@
 import logging
 import re
 
+from email import message
```

Comment:
I think this one might be my fault from the register test haha. VSCode likes auto-importing things 😓 
```suggestion
```